linear-solver-spec.json

```diff
@@ -153,6 +155,16 @@
         ],
         "doc": "Settings for the AMGCL solver."
     },
+    {
```

Comment:
i dont think you need any of these, there are no options for trilinos

Reply:
you need to add the options from trilinos

Reply:
Fixed.